pages/install/fedora-rhel-install.rst

```diff
@@ -0,0 +1,58 @@
+.. title: Installing Cantera on Fedora
+.. date: 2021-10-19 17:49:00 UTC+03:00
+.. description: Installation instructions for Cantera on Fedora
+.. type: text
+.. _sec-install-fedora-rhel:
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Installing on Fedora</h1>
+
+   .. class:: lead
+
+      RPM packages are provided for supported versions of Fedora Linux using a Community Projects
+      (COPR) repository. Note that the Matlab interface is not available from this archive;
+      to install the Matlab interface on Fedora, you must :ref:`compile the source code <sec-compiling>`.
+
+As of Cantera 2.6.0, packages are available for Fedora Linux 33, 34, 35, and
```

Comment:
Since I made the PR, F33 is no longer supported.
I'll update the language appropriately